app/src/main/java/com/framgia/forder/screen/shopinfo/ShopinfoPresenter.java

```diff
@@ -4,9 +4,14 @@
 import com.framgia.forder.data.source.ShopRepository;
 import com.framgia.forder.data.source.remote.api.error.BaseException;
 import com.framgia.forder.data.source.remote.api.error.SafetyError;
+import com.framgia.forder.data.source.remote.api.request.ApplyShopToDomainRequest;
+import com.framgia.forder.data.source.remote.api.request.LeaveShopToDomainRequest;
+import com.framgia.forder.data.source.remote.api.response.RegisterShopResponse;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.source.remote.api.response.RegisterShopResponse. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports